gradle/dependencies.gradle

```diff
@@ -15,367 +15,386 @@
+                    compile('org.ow2.asm:asm-util') { version { strictly versions.asm } }
+                    compile('com.esotericsoftware.minlog:minlog') { version { strictly '1.2' } }
+                    compile('org.sonatype.aether:aether-api') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-impl') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-spi') { version { strictly versions.aether } }
+                    compile('org.sonatype.aether:aether-util') { version { strictly versions.aether } }
+                    compile('com.google.api-client:google-api-client') { version { strictly '1.23.0' } }
+                    compile('com.google.http-client:google-http-client') { version { strictly '1.23.0' } }
+                    compile('com.google.http-client:google-http-client-jackson2') { version { strictly '1.23.0' } }
+                    compile('com.google.oauth-client:google-oauth-client') { version { strictly '1.23.0' } }
+                    compile('org.hamcrest:hamcrest-core') { version { strictly '1.3' } }
+                    compile('org.apache.httpcomponents:httpcore') { version { strictly '4.4.4' } }
+                    compile('com.beust:jcommander') { version { strictly '1.47' } }
+                    compile('org.apache.maven:maven-aether-provider') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-artifact') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-compat') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-model') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-model-builder') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-plugin-api') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-repository-metadata') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-settings') { version { strictly versions.maven } }
+                    compile('org.apache.maven:maven-settings-builder') { version { strictly versions.maven } }
+                    compile('org.sonatype.plexus:plexus-cipher') { version { strictly '1.7' } }
+                    compile('org.codehaus.plexus:plexus-classworlds') { version { strictly '2.4' } }
+                    compile('org.codehaus.plexus:plexus-component-annotations') { version { strictly '1.5.5' } }
+                    compile('org.codehaus.plexus:plexus-interpolation') { version { strictly '1.14' } }
+                    compile('org.codehaus.plexus:plexus-sec-dispatcher') { version { strictly '1.3' } }
+                    compile('org.codehaus.plexus:plexus-utils') { version { strictly '3.0.8' } }
```

Comment:
We should verify that we really want all of these different versions from `plexus`, including one from `org.sonatype.plexus` and others from `org.codehaus.plexus`!